### ISSUE TITLE
Should the accessibility help work out of editor?

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#108850]
Hello,
It seems you can trigger the show accessibility help command only from an active editor.
Considering this accessibility support is mostly about an editor, it actually makes sense. However, in case of linux it's a little special because
if you run vscode without the --force-renderer-accessibility flag and without editor accessibilty explicitly enabled, vscode is completely silent and inaccessible. It is of course an expected behavior at least until chrome is able to detect a screenreader, however in that case, in order to enable accessibility, before pressing the show accessibility help keystroke you have to be in an editor, so you have to press ctrl+n... And it is a bit weird.

**Comment by isidorn** [comment]
@webczat I have pushed a change for this. You can try it out with vscode insiders from Wednesday and let us know how it works for you.

**Comment by webczat** [comment]
hmmm, I downloaded the insiders just now, pressed alt+shift+1 on a window with closed editors, nothing happens. either insiders somehow too old, or this didn't work?

**Comment by isidorn** [comment]
You are correct, I made a mistake.
The issue is that the AccessibilityHelp Widget is designed in such a way that it requires an Editor - since it is an overlay widget. So we simply can not show it if there is no editor around.

As a simple workaround I can bind a new command to the same keybinding that would simply open a new empty editor and trigger the old command.
Let me reopen this issue

**Comment by isidorn** [comment]
I have pushed a fix for this. To verify:
1. Have an editor open and press Alt + F1 > Accessibility help should be shown
2. Have an editor open but not focues. Alt + F1 > Accessibility help should be shown
3. Have 0 editors open. Alt + F1 > New untitiled editor should be opened and accessibility help should be shown

**Comment by webczat** [comment]
honestly wondering if it's better to fix it like this, or just leave it as is, but say in docs that you shall press ctrl+n before using the shortcut because editor. as in, it seems weird to someone without the context that pressing this opens the editor, even though it seems to be the only way to do it. but here I'd ask others... what is your opinion?

**Comment by isidorn** [comment]
I think that Alt + F1 should always open the accessibility help, no matter the state of the workbench.
Same way that F1 always opens the command palette no matter the state of the workbench.

So let's go with this approach and see if somebody finds it unexpected.
Thanks

**Comment by webczat** [comment]
well, I would find it weird myself, but it's just me. let's do as you suggest, when I'll have a chance I'll try to verify.

**Comment by webczat** [comment]
oh sorry. just verified it indeed works, forgot I should do this.

**Comment by isidorn** [comment]
@webczat thanks for letting us know!
